ftw/shop/browser/templates/cart_edit.pt

```diff
@@ -63,10 +63,12 @@
                       <td>
                           <a href="" tal:attributes="href python:'cart_remove?skuCode=%s' % skuCode"
                                      i18n:attributes="cart_remove">
-                              <img src="++resource++ftw-shop-resources/delete_p.gif" />
+                              <img tal:define="portal_url context/@@plone_portal_state/portal_url"
+                                  tal:attributes="src string:${portal_url}/++resource++ftw-shop-resources/delete_p.gif"
+                                src="++resource++ftw-shop-resources/delete_p.gif" />
```

Comment:
I'd remove the static `src`-Attribute here, because it is filled by `tal:attributes` and therefore not necessary / confusing :wink: 
There are some other places with the same duplication.